_data/education.yml

```diff
@@ -13,6 +13,14 @@ websites:
       software: Yes
       doc: https://support.clever.com/hc/en-us/articles/202062333-Two-Factor-Authentication
 
+    - name: CSN (The Swedish National Board of Student Aid)
```

Comment:
Please change the name to just `CSN` as CSN says themselves that:
```
CSN's activities are broken down into two areas, each of which reports to a separate Government Ministry:
* Financial Aid for Studies (Ministry of Education and Research)
* Home Equipment Loans (Ministry of Employment).
```
More information about Home Equipment Loans can be found here: http://www.csn.se/hemutrustningslan
_Source: http://www.csn.se/en/2.1076/2.1077_